app/views/application/_header.html.haml

```diff
@@ -1,32 +1,32 @@
 %header
-  %div{:class => "navbar navbar-expand-lg navbar-dark bg-dark static-top"}
-    %div{:class => "container-fluid"}
-      %div{:class => "navbar-header"}
-        = link_to discovery_path, id: "logo", class: "navbar-brand w-50" do
-          = image_tag "logo450x429.png", id: "logoheader"
-      %button{:type => "button", :class => "navbar-toggler", :data => {:toggle => "collapse", :target => "#navbarResponsive"}, :aria => {:expanded => "false", :controls => "navbarResponsive", :label => "Toggle navigation"}}
-        %span{:class => "navbar-toggler-icon"}
-      %div{:id => "navbarResponsive", :class => "collapse navbar-collapse w-100"}
-        %ul{:class => "navbar-nav w-100"}
+  .navbar.navbar-expand-lg.navbar-dark.bg-dark.static-top
```

Comment:
Is there a way to limit this in any way? It's a lot of classes for one line

Reply:
These classes have always been here. We just rearranged them differently. I am not sure if there are ways to shorten this line, but that is not the purpose of this PR.